README.md

```diff
@@ -12,7 +12,8 @@ The Current LTS Plan is:
    the end of August, 2015, with the first LTS release cut during the first
    week of October, 2015.
 2. After that, new LTS releases will occur once every 12 months, at the same
-   time each year.
+   time each year (October 1), based on the even-major-number release cut in
+   April (e.g. 8.0 is released in April, 8.3LTS is released in October).
```

Comment:
`8.3 LTS` `could be released` -- Let's try to not firmly say it will be 8.3 in particular.